src/scripts/records/ModalRecord.js

```diff
@@ -0,0 +1,15 @@
+/* @flow */
+
+import { Record as ImmutableRecord } from 'immutable'
+
+class ModalRecord extends ImmutableRecord({
+  modalContent: '',
+  modalCallback: () => null,
+  showModal: false
+}) {
+  modalContent: string
+  modalCallback: () => null
```

Comment:
I don't think we want to enforce a type of `() => null`. Maybe just `() => any`? 

`() => void` if we do not want any return values

Reply:
Done https://github.com/Paratii-Video/paratii-portal/pull/301/commits/ef56e9a93bb88ac25241e85ce446283d1bd98d44